site/docs/components/range-slider/examples.mdx

```diff
@@ -105,6 +111,16 @@ This approach ensures a user-friendly and accessible interface for selecting non
 
 </LivePreview>
 
+<LivePreview componentName="range-slider" exampleName="WithRestrictedLabelOverflow">
+
+## With Restricted Label Overflow
```

Comment:
Labels had examples above and then we comeback to labels again at the end of the page, can we combine this description into consecutive sections, so that everything is in one place ?